coral-common/src/main/java/com/linkedin/coral/common/transformers/JsonTransformSqlCallTransformer.java

```diff
@@ -50,6 +50,8 @@ public class JsonTransformSqlCallTransformer extends SourceOperatorMatchSqlCallT
     OP_MAP.put("/", SqlStdOperatorTable.DIVIDE);
     OP_MAP.put("^", SqlStdOperatorTable.POWER);
     OP_MAP.put("%", SqlStdOperatorTable.MOD);
+    OP_MAP.put("greatest", new SqlUserDefinedFunction(new SqlIdentifier("greatest", SqlParserPos.ZERO),
+        ReturnTypes.ARG0_NULLABLE, null, OperandTypes.SAME_VARIADIC, null, null));
```

Comment:
I feel it is simpler to use `CASE WHEN` compared to `greatest`.

Reply:
addressed in SubstrIndexTransformer.